frontend/trip-planner/src/Components/EmailAdd.js

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { ReactMultiEmail, isEmail } from 'react-multi-email';
+import 'react-multi-email/dist/style.css';
+
+function Email(props) {
+  //const [emails, setEmails] = useState([]);
+  const [focused, setFocused] = useState(false);
```

Comment:
`focused` is not used anywhere, I think you can remove this line along with lines 17-18